microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md

```diff
@@ -64,19 +64,19 @@ To edit (or define) ATP policies, you must be assigned an appropriate role. The
 
 - Any URLs that you specify in the "do not rewrite" list are excluded from ATP Safe Links scanning for the recipients that you specify.
  
-- If you already have a list of URLs in your "do not rewrite" list, make sure to review that list and add wildcards as appropriate. For example, if your existing list has an entry like `http://contoso.com/a` and you want to include subpaths like `http://contoso.com/a/b` in your policy, add a wildcard to your entry so it looks like `http://contoso.com/a*`.
+- If you already have a list of URLs in your "do not rewrite" list, make sure to review that list and add wildcards as appropriate. For example, if your existing list has an entry like `http://contoso.com/a` and you want to include subpaths like `http://contoso.com/a/b` in your policy, add a wildcard to your entry so it looks like `http://contoso.com/a/*`.
     
 - Do not include a forward slash (/) in the URLs that you specify in your "do not rewrite" list. For example, rather than enter `contoso.com/` in your "do not rewrite" list, enter `contoso.com`.
 
-- When you specify a "do not rewrite" list for an ATP Safe Links policy, you can include up to three wildcard asterisks (\*). Wildcards (\*) are used to explicitly include prefixes or subdomains, like `http://` or `https://`. An entry, such as `contoso.com` is not the same as `*contoso.com*` for your "do not rewrite" list. You must have `*contoso.com*` if you want to allow people to visit a domain and its subdomains and paths.
+- When you specify a "do not rewrite" list for an ATP Safe Links policy, you can include up to three wildcard asterisks (\*). Wildcards (\*) are used to explicitly include prefixes or subdomains, like `http://` or `https://`. An entry, such as `contoso.com` is not the same as `*.contoso.com/*` for your "do not rewrite" list. You must have `*.contoso.com/*` if you want to allow people to visit a domain and its subdomains and paths.
     
 The following table lists examples of what you can enter and what effect those entries have.
     
 |**Example Entry**|**What It Does**|
 |:-----|:-----|
 |`contoso.com`|Allows recipients to visit a site like `http://contoso.com` but not subdomains or paths.|
-|`*contoso.com*`  <br/> |Allows recipients to visit a domain, subdomains, and paths, such as `http://www.contoso.com`, `https://www.contoso.com`, `https://maps.contoso.com`, or `http://www.contoso.com/a`  <br/> |
+|`*.contoso.com/*`  <br/> |Allows recipients to visit a domain, subdomains, and paths, such as `http://www.contoso.com`, `https://www.contoso.com`, `https://maps.contoso.com`, or `http://www.contoso.com/a`  <br/> |
```

Comment:
I don't think this example still allows https://maps.contoso.com (because that doesn't have a / at the end).

Reply:
Note - in my issue, I wasn't suggesting you remove the "`*contoso.com*` example, but add another *.contoso.com/* example, with the "what it does" saying something like "Allows recipients to visit legitimate sites in the contoso.com domain like http://accounts.contoso.com/a or http://contoso.com/a/b but not fraudulent sites seeking to bypass the rule, like http://www.fake.contoso.completelyfake.com."